clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+* Properly configured trust store on client system
+
+In order for the producer to verify the certificate of the brokers make sure the trust store is set up correctly on your system. 
+
+** CentOS
+
+```bash
+$ sudo yum reinstall ca-certificates
+```
+
+Add the following property to the `Producer` and `AdminClient` constructors in `producer.py`:
+
+```
+ssl.ca.location='/etc/pki/tls/cert.pem'
+```
+
+or symlink or copy `/usr/ssl/cert.pem` to `/etc/pki/tls/cert.pem`. This is the default location used by the python producer. For more information see the librdkafka docs on which this python producer is built: https://github.com/edenhill/librdkafka/wiki/Using-SSL-with-librdkafka
```

Comment:
I don't think we should recommend copying these files around since it might interfere with the package manager. Explicitly specifying the path of the CA cert if it is in a non-standard location is better.

Reply:
Ok I will remove that part.

Reply:
Fixed https://github.com/confluentinc/examples/pull/144/commits/ad8efa1f74a377c1d596cb6c0323830a22e42a54